demos/basics/loading-numpy.ipynb

```diff
@@ -0,0 +1,923 @@
+{
```

Comment:
<p>"This notebook walks through loading two kinds of graphs" lists 3 bullet points below?</p><p> </p><p>I think it should be "This notebook walks through loading three kinds of graphs:"</p><p> </p><p>Also there's two different paragraphs talking about NumPy vs Pandas - could they be combined?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1599/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Could you be a bit more specific about which paragraphs you're talking about? There's the first one, the "StellarGraph supports loading data from ..." one and the "This notebook only uses NumPy for the node features ...", and all of them touch on this in some form.</p>